### ISSUE TITLE
Releases tab misssing

### ISSUE THREAD

**Issue by probonopd** [isaacs/github#1825]
Releases tab misssing.
Where did it go?

![releases](https://user-images.githubusercontent.com/2480569/85623158-7fc13580-b657-11ea-8269-5d09395a7bd2.png)

**Comment by Sigma88** [comment]
I would also like to get the releases tab back

**Comment by treier57lol** [comment]
Releases have been moved to the side bar.
![Screenshot_20200626_133323](https://user-images.githubusercontent.com/33978534/85848475-dcd0fe80-b7b1-11ea-98d0-7d4dffde7beb.png)

**Comment by Sigma88** [comment]
that button is only available in the home page, while the tab was always available.

Also, if there are pre-releases but no "full" releases, that button will not show up.

to me this seems definitely a downgrade from the previous system

**Comment by treier57lol** [comment]
Wait, releases didn't even have a seperate tab at the top, it was only in the code tab as a header item:
![image](https://user-images.githubusercontent.com/33978534/85880133-2c302280-b7e4-11ea-96a9-1cac42c8ea4f.png)
![image](https://user-images.githubusercontent.com/33978534/85880668-140cd300-b7e5-11ea-8695-d9c0fecca000.png)
But yes, there is no button when there are only pre-releases.

**Comment by probonopd** [comment]
Exactly, I always clicked on "Releases" to see whether there were Pre-Releases (a.k.a. continuous builds) available.

**Comment by treier57lol** [comment]
Wow, nevermind that, it's already fixed. Even with only pre-releases, the releases button is available.
![Screenshot_20200627_151040](https://user-images.githubusercontent.com/33978534/85921958-98fcf880-b888-11ea-8c3a-7639a6f1637c.png)
But still, there was no seperate tab for releases to my knowledge, only a button in the code tab.

**Comment by probonopd** [comment]
Have they changed it since a few days ago? I couldn't find it, but today I do...

**Comment by treier57lol** [comment]
Yes, with pre-releases the button was missing yesterday but now it's fixed.

**Comment by janbrasna** [comment]
There was never a "Releases tab" that would be reachable from e. g. Issues or Wiki, it has always been only a part of "Code" scope navigation items among commits and branches…
